vnext/JSI/Desktop/ChakraJsiRuntime_core.cpp

```diff
@@ -320,15 +226,6 @@ JsErrorCode ChakraRuntime::enableDebugging(
   return result;
 }
 
-/* static */ void ChakraRuntime::ProcessDebuggerCommandQueueCallback(
-    void *callbackState) {
-  ChakraRuntime *runtime = reinterpret_cast<ChakraRuntime *>(callbackState);
-
-  if (runtime) {
-    runtime->ProcessDebuggerCommandQueue();
-  }
-}
-
 void ChakraRuntime::ProcessDebuggerCommandQueue() {
   if (runtimeArgs().jsQueue) {
     runtimeArgs().jsQueue->runOnQueue([this]() {
```

Comment:
>this [](start = 39, length = 4)

who is ensuring that this stays alive when queue is executed?

Reply:
No one. We need to fix this, but that belongs to a separate PR.

---
In reply to: [333114231](https://github.com/microsoft/react-native-windows/pull/3366#discussion_r333114231) [](ancestors = 333114231)